nav2_bt_navigator/src/bt_navigator.cpp

```diff
@@ -126,28 +128,70 @@ BtNavigator::on_configure(const rclcpp_lifecycle::State & /*state*/)
   blackboard_->set<int>("number_recoveries", 0);  // NOLINT
 
   // Get the BT filename to use from the node parameter
-  std::string bt_xml_filename;
-  get_parameter("bt_xml_filename", bt_xml_filename);
+  get_parameter("bt_xml_filenames", bt_xml_filenames_);
+  get_parameter("bt_xml_ids", bt_xml_ids_);
+
+  if (bt_xml_filenames_.empty()) {
+    RCLCPP_ERROR(get_logger(), "Couldn't find any XML file");
+    return nav2_util::CallbackReturn::FAILURE;
+  }
+
+  if (!loadBehaviorTree(bt_xml_ids_[0])) {
```

Comment:
Why load the first? I don't think that's a promise to be the default tree?

Reply:
Now the default BT is explicitly specified